bin/merge.sh

```diff
@@ -124,38 +125,33 @@ merge_xdev_into_sdev () {
 }
 
 
-check_cv32e40x_repo() {
+clone_x_dv() {
```

Comment:
The S->X merge now starts with cloning x-dv.
It asks for y/n confirmation first.

I think this streamlines the most typical use case of this script.
Please let me know if I overlooked any reason this might be a bad idea.

Reply:
Nice!